lib/generators/open_stax/rescue_from/install/templates/rescue_from.rb

```diff
@@ -0,0 +1,51 @@
+require 'openstax_rescue_from'
+
+OpenStax::RescueFrom.configure do |config|
+  config.raise_exceptions = ![false, 'false'].include?(ENV['RAISE_EXCEPTIONS']) ||
```

Comment:
Nice catch.  Similar to https://github.com/openstax/tutor-server/blob/master/lib/env_utilities.rb#L3